# Define the goal
std_msgs/Header header
int32 control
---
# Define the result
std_msgs/Header header
bool done
float32 pole_angle
float32 pole_velocity
float32 cart_position
float32 cart_velocity
float32 reward
---
# Define a feedback message 
int32 error_code

